network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -43,6 +43,24 @@
 
 vote_throttle_class = UserVoteRateThrottle if not settings.TESTING else TestUserVoteRateThrottle
 
+# This is a hardcoded date that we update every time we release a new version of PNI
+public_cutoff_date = datetime(2020, 10, 29)
```

Comment:
Let's set it to a config or env variable eventually. (not right now)

Reply:
filed https://github.com/mozilla/foundation.mozilla.org/issues/5854 to track